raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.tsx

```
@@ -1,42 +1,46 @@
-import React, { ReactNode } from "react";
+import React, { ReactNode, useState } from "react";
```

ESLint

[error] 'useState' is defined but never used.

(@typescript-eslint/no-unused-vars)